package.json

```diff
@@ -22,21 +22,21 @@
   "license": "MIT",
   "version": "0.2.0",
   "engines": {
-    "node": ">=6.0.0"
+    "node": ">=8.9.0"
   },
   "dependencies": {
     "lodash": "4.x.x",
     "prettyoutput": "1.x.x",
     "uuid": "3.x.x"
   },
   "devDependencies": {
-    "ava": "0.21.0",
-    "coveralls": "^2.13.1",
-    "eslint": "4.4.1",
+    "ava": "1.0.0-rc.2",
+    "coveralls": "2.x.x",
```

Comment:
Should be `2.x`, the same applies for other packages using this notation

Reply:
Updated